client/sites/components/sites-dashboard.tsx

```diff
@@ -74,10 +75,9 @@ const siteSortingKeys = [
 const DEFAULT_PER_PAGE = 50;
 const DEFAULT_SITE_TYPE = 'non-p2';
 
-// Limit fields on breakpoints smaller than 960px wide.
-const desktopFields = [ 'site', 'plan', 'status', 'last-publish', 'stats' ];
-const mobileFields = [ 'site' ];
-const listViewFields = [ 'site' ];
+const desktopFields = [ 'icon', 'site', 'title-text', 'plan', 'status', 'last-publish', 'stats' ];
+const mobileFields = [ 'icon', 'title-text', 'site-url', 'admin-url' ];
+const listViewFields = [ 'title-text', 'site-url' ];
```

Comment:
Not sure about how these are working, something feels a bit broken based on my testing.